app/views/vzory/stranky-so-spatnou-vazbou.njk

```diff
@@ -50,7 +50,7 @@
           <li>názvu použitej služby (aby používateľ vedel, ku ktorej službe zasiela spätnú väzbu),</li>
           <li>otázky: “Ako ste spokojný/á so službou, ktorú ste práve využili?”,</li>
           <li>prepínačov s možnosťami: Veľmi spokojný/á, Nespokojný/á, Ani spokojný/á ani nespokojný/á, Spokojný/á a Veľmi spokojný/á,</li>
-          <li>textového poľa s otázkou “Ako by sme mohli túto službu zlepšiť?”, ktoré sa zobrazí pri výbere jednej z nasledujúcich možností: Veľmi spokojný/á, Nespokojný/á a Ani spokojný/á ani nespokojný/á.</li>
+          <li>textového poľa s otázkou “Ako by sme mohli túto službu zlepšiť?”, ktoré sa zobrazí pri výbere jednej z nasledujúcich možností: Veľmi nespokojný/á, Nespokojný/á a Ani spokojný/á ani nespokojný/á.</li>
```

Comment:
Cauko Feri,

si si isty ze to tak ma byt? potom tam nie je kladnej moznosti :) len velmi nespokojny (zaporny), nespokojny (zaporny) a ani spokojny ani neskopojny (neutralny).. 

teraz to ja chapem takto ze je velmi spokojny (kladny)/velmi nespokojny (zaporny) a ani spokojny ani nespokojny (neutralny)